ibm_vpd_app.cpp

```diff
@@ -1246,7 +1246,8 @@ int main(int argc, char** argv)
 
         baseFruInventoryPath = js["frus"][file][0]["inventoryPath"];
         // Check if we can read the VPD file based on the power state
```

Comment:
does this comment need change, I am ok if not also , just asked since the check enhanced